data/presets/amenity/baking_oven/bakehouse.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "temaki-bread",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "baking_oven",
+        "building": "Bakehouse"
```

Comment:
```suggestion
        "building": "bakehouse"
```

Reply:
Thank You so much for reviewing...I have committed your suggestion...Hope it gets merged...